cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -40,30 +40,48 @@
           "widget-attributes": {
             "values": [
               "auto-detect",
+              "africa-south1",
               "asia-east1",
               "asia-east2",
               "asia-northeast1",
               "asia-northeast2",
               "asia-northeast3",
               "asia-south1",
+              "asia-south2",
               "asia-southeast1",
               "asia-southeast2",
               "australia-southeast1",
+              "australia-southeast2",
+              "europe-central2",
```

Comment:
Is it okay, according to Horizontals, if we run the pipelines on Dataproc clusters outside of the CDF instance location? I understand that it is ephemeral but just wanted to know.